save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/utils/Warning.kt

```diff
@@ -57,6 +57,32 @@ internal fun String.extractWarning(warningRegex: Regex,
     )
 }
 
+/**
+ * @param warningRegex regular expression for warning
+ * @param fileName file name
+ * @param line line number of warning
+ * @param message waring text
+ * @param columnGroupIdx index of capture group for column number
+ * @return a [Warning] or null if [this] string doesn't match [warningRegex]
+ */
+@Suppress("TOO_MANY_PARAMETERS", "LongParameterList")
```

Comment:
Do we need to use `TOO_MANY_PARAMETERTS` suppress if we have 5 args?

Reply:
Remove `TOO_MANY_PARAMETETRS` suppress pls (if it isn't needed here)

Reply:
You're right. Fixed